### PR TITLE
Feature/gs/jl/db connection

### PR DESCRIPTION
**Pull request by gursham2001**

### CODE REVIEW

File: src/main/java/com/sparta/om/App.java [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 public class App 
 {
-    public static void main( String[] args )
-    {
-        EmployeeDAO.PopulateArray("src/main/resources/EmployeeRecords.csv");
+    public static void main(String[] args) {
+        Connection postgresConnection = ConnectionManager.connectToDB();
+        UserDAO userDAO = new UserDAO(postgresConnection);
+
+        userDAO.printAllUsers();
```
Comment (JamalLindo) [inline src/main/java/com/sparta/om/App.java]: No longer need to print all users ?

File: src/main/java/com/sparta/om/DB/controller/ConnectionManager.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionManager {
+    private static Connection postGresConnection;
+
+    public static Connection connectToDB() {
+        String url = PropertiesLoader.getProperty("url");
+        String username = PropertiesLoader.getProperty("userName");
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/DB/controller/ConnectionManager.java]: Make "userName" and "passWord" lowercase

File: src/main/java/com/sparta/om/DB/controller/ConnectionManager.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.sparta.om.DB.controller;
+
+import com.sparta.om.DB.util.PropertiesLoader;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionManager {
+    private static Connection postGresConnection;
```
Comment (roshnihpatel) [inline src/main/java/com/sparta/om/DB/controller/ConnectionManager.java:10]: postGresConnection not following naming convention, should be **postgresConnection**

File: src/main/java/com/sparta/om/DB/model/SQLQueries.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.sparta.om.DB.model;
+
+public interface SQLQueries {
+
+    public static final String SELECT_ALL = "SELECT * FROM public.user_db";
+
+    String INSERT_INTO_DB = "INSERT INTO public.user_db (id, name, email) Values (?,?,?)";
```
Comment (JamalLindo) [inline src/main/java/com/sparta/om/DB/model/SQLQueries.java]: Values should be VALUES

Comment (gursham2001) [inline src/main/java/com/sparta/om/DB/model/SQLQueries.java]: There should be question marks after the Question Marks

File: src/main/java/com/sparta/om/DB/model/UserDAO.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class UserDAO {
+
+    private final Connection postgresConnection;
+
+    private Statement statement;
+
+    public UserDAO(Connection postgresConnection) {
```
Comment (JamalLindo) [inline src/main/java/com/sparta/om/DB/model/UserDAO.java:13]: Method should be lowercase start

File: src/main/java/com/sparta/om/DB/util/PropertiesLoader.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    public static String getProperty(String key) {
+        Properties properties = new Properties();
+        try {
+            properties.load(new FileReader("src/main/resources/db.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return properties.getProperty(key);
+    }
+
```
Comment (gursham2001) [inline src/main/java/com/sparta/om/DB/util/PropertiesLoader.java]: remove white space

Comment (JamalLindo) [inline src/main/java/com/sparta/om/DB/util/PropertiesLoader.java:18]: White space

**Review by JamalLindo — APPROVED** [review]

**Pull request merged**